### PR TITLE
Version update and repository links

### PR DESCRIPTION
**Pull request by niklabh**

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This library provides additional typing information for user to access Astar's m
 
 # Getting Started
 
-More documentation and examples on [wiki](https://github.com/webb-tools/astar.js/wiki)
+More documentation and examples on [wiki](https://github.com/astarNetwork/astar.js/wiki)
```
Comment (hoonsubin) [inline README.md:8]: should be `AstarNetwork`

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {
-  "bugs": "https://github.com/webb/api/issues",
+  "bugs": "https://github.com/astarNetwork/astar.js/issues",
```
Comment (hoonsubin) [inline package.json:2]: same as above

File: packages/api-derive/package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "@astar-network/astar-api-derive",
-  "version": "0.1.5-7",
+  "version": "0.1.8",
   "description": "Astar JS API",
   "main": "index.js",
-  "author": "Webb Developers <drew@webb.tools>",
+  "author": "Astar Network <nikhil@astar.network>",
```
Comment (hoonsubin) [inline packages/api-derive/package.json:6]: I think you can use `info@astar.network`

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (niklabh) [reply README.md:8]: Fixed here https://github.com/AstarNetwork/astar.js/pull/25

File: packages/api-derive/package.json (hunk 3 above)
Comment (niklabh) [reply packages/api-derive/package.json:6]: Fixed in https://github.com/AstarNetwork/astar.js/pull/25

**Pull request merged**